src/services/drpcEvents/index.ts

```diff
@@ -241,4 +249,85 @@ export default class DrpcEvents {
       })
     }
   }
+
+  private async handleParentQuery(id: UUID, childQuery: QueryRow, params: SubmitQueryResponseRPCParams) {
+    const [parentQuery]: QueryRow[] = await this.db.get('query', { id })
+    if (!parentQuery) {
```

Comment:
we should check that we haven't already respponded to the parent query and error if we have

Reply:
this also has been included right after